server/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
remember to also include the package-lock.json in here

Reply:
Yeah this is something I'm never sure about since `node ci` uses package-lock.json to do a clean install so I feel it is useful to add, but probably not worth it